### PR TITLE
Changes made during and after colorado

### PR DESCRIPTION
**Pull request by nobody5050**

### CODE REVIEW

File: src/main/cpp/Robot.cpp [hunk 1]
```diff
@@ -78,6 +78,7 @@ void Robot::TeleopInit() {
   }
 
   m_container.StartIdling();
+  m_container.StopMotors();
   // m_container.ResetPose();
```
Comment (having11) [inline src/main/cpp/Robot.cpp:82]: Still want this?

File: src/main/include/Constants.h [hunk 2]
```diff
@@ -186,14 +186,15 @@ extern const frc::TrapezoidProfile<units::degree>::Constraints
 
 const std::string kDefaultAutoName = "Leave Wing";
 
-const std::string kScoreSubwooferName = "Shoot Subwoofer";
+const std::string kScoreSubwooferName = "Score Subwoofer";
```
Comment (having11) [inline src/main/include/Constants.h:189]: Can we make these less ambiguous?

File: src/main/include/Constants.h [hunk 3]
```diff
@@ -362,7 +366,7 @@ constexpr double kSpeakerUpperSpeed = kSpeakerLowerSpeed;
 
 // These should also match
 // TODO: CHANGE TO VELOCITY RATHER THAN % OUTPUT
-constexpr double kSubwooferLowerSpeed = -0.75;
+constexpr double kSubwooferLowerSpeed = -0.95;
```
Comment (having11) [inline src/main/include/Constants.h:369]: Does this need to go back down?

File: src/main/include/utils/CommandUtils.h [hunk 4]
```diff
@@ -38,6 +40,9 @@ static frc2::CommandPtr ArmScoringGoal(
                  // TODO: MAKE THIS A CONSTANT
                  return arm->MoveToPositionAbsolute(142_deg);
                } else {
+                 // // In case it got stuck up, make it go down when not
```
Comment (having11) [inline src/main/include/utils/CommandUtils.h:43]: Still want these comments?

File: src/main/include/utils/CommandUtils.h [hunk 5]
```diff
@@ -51,13 +56,51 @@ static frc2::CommandPtr ScoreRamp(std::function<ScoringDirection()> direction,
...
+}
+
+static frc2::CommandPtr PreScoreShuffle(
+    std::function<ScoringDirection()> direction, ScoringSubsystem* scoring,
+    IntakeSubsystem* intake) {
+  return frc2::InstantCommand([] {
+           ConsoleLogger::getInstance().logInfo("Scoring", "Pre Score Shuffle");
+         })
+      .ToPtr()
+      .AndThen((frc2::InstantCommand([scoring, intake, direction] {
```
Comment (having11) [inline src/main/include/utils/CommandUtils.h:87]: I would turn this into a `FunctionalCommand`

**Review by having11 — CHANGES_REQUESTED** [review]

File: src/main/cpp/RobotContainer.cpp [hunk 6]
```diff
@@ -82,7 +86,7 @@ void RobotContainer::RegisterAutos() {
         ConsoleLogger::getInstance().logVerbose("Autos", "Intaking %s", "");
       })
           .ToPtr()
-          .AndThen(IntakingCommands::Intake2(&m_intake, &m_scoring)));
+          .AndThen(IntakingCommands::Intake3(&m_intake, &m_scoring)));
```
[the remark this answers is not in the document]
Comment (nobody5050) [reply src/main/cpp/RobotContainer.cpp:89]: I would like to refactor intake 2 to behave like intake 3 does, and then switch to that

File: src/main/cpp/Robot.cpp (hunk 1 above)
Comment (nobody5050) [reply src/main/cpp/Robot.cpp:82]: Yes, if the auto doesn’t finish in 15s it needs to stop in teleop init

Comment (having11) [reply src/main/cpp/Robot.cpp:82]: Was referring to the commented-out call to `ResetPose`

File: src/main/include/Constants.h [hunk 7]
```diff
@@ -293,8 +293,14 @@ constexpr double kDriveDeadband = 0.05;
 constexpr double kVibrationIntensity = 1;
 }  // namespace OIConstants
 
+// LED Constants
 constexpr uint8_t kLedAddress = 23;
 constexpr units::second_t kConnectorXDelay = 0.002_s;
+constexpr double kAccelThreshold = 7;
+
+// BaseSingleAxisSubsystem Constants
+constexpr double kMotorDeadSpeedRange = 0.05;
```
Comment (WowMuchDoge) [inline src/main/include/Constants.h:302]: Put this in its own namespace

**Review by WowMuchDoge — CHANGES_REQUESTED** [review]

**Review by WowMuchDoge — APPROVED** [review]

**Review by having11 — APPROVED** [review]
LGTM. Nice job and thanks for using constants.

**Pull request merged**